homeassistant/components/overkiz/sensor.py

```diff
@@ -182,6 +182,13 @@ class OverkizSensorDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfPower.WATT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    OverkizSensorDescription(
```

Comment:
Can you do these sensor (and preferably binary_sensor) changes in a separate PR? It is preferred to only change one entity platform per PR.

These changes can be merged very quick, no need to wait as long as for this PR.

Reply:
done. I will create a PR as soon as this one is merged.